### DISCUSSION TITLE
Automated 'export'/'send to' from Qfield/QfieldCloud - development request

### DISCUSSION THREAD

**Discussion by RatInTheField** [opengisch/QField#5609] [Q&A]
I have been using Qfield for some years now in a field data collection role. I have a number of tablets operated by field surveyors who collect feature data from the field and return it to base to be merged into a master database.
For a long while, this method worked well using Android 10, but recently, I have had to invest in new tablets that were loaded with Android13 and have had to change my workflow to suit Google's dictates.
To help get around Googles changes, I export data from Qfield by clicking the three dots relating to a file, then follow the options to export or send the file to a location. This works well as a one off, but when you are doing it several times a day for a number of devices, it gets tedious!

Would it be possible to automate the export/Send to routine so that it can be run automatically to a schedule?

**Comment by nirvn** [comment]
@RatInTheField , good news, the automated regular push of changes to QFieldCloud has been implemented for a while now (since QField 3.2.0). You can enable this functionality and set the push interval in the QField panel when configuring your project with qfieldsync plugin:

![image](https://github.com/user-attachments/assets/710d776f-4fd6-43b8-8a5e-69c163d94275)

**Comment by RatInTheField** [comment]
Hi Mattieu,

 

Thanks for getting back to me. I think that I did not explain myself well
enough. I am aware of the ability to automate the push function to
QFieldCloud, but would like to be able to automate the Send To function
accessed from QField Menu>Cog>Project Folder>3 dots >'Send to' without using
QFieldCloud.

 

I want to be able to send files to a remote location managed by another app
such as FV File Explorer so that I can send data to my own SSH protected
server. While this is currently possible as an ad-hoc function, repeating
the action many times a day would be boring and risk errors, so I was hoping
that it would be possible to automate and schedule the Send To function to
speed my workflow.

 

Is this possible? Could it be within your design scope?

 

Roland Owen

 

 ***@***.***> ***@***.***

07530763074

**Comment by m-kuhn** [comment]
@RatInTheField I assume this should be doable with a plugin, calling `platformUtilities.sendCompressedFolderTo(qgisProject.homePath)` at regular intervals or a prominent button.

**Comment by RatInTheField** [comment]
Thanks for the suggestion. I'll try that later.
